gen/src/write.rs

```diff
@@ -337,7 +349,11 @@ fn write_opaque_type<'a>(out: &mut OutFile<'a>, ety: &'a ExternType, methods: &[
     writeln!(out, "#ifndef {}", guard);
     writeln!(out, "#define {}", guard);
     for line in ety.doc.to_string().lines() {
-        writeln!(out, "//{}", line);
+        if cfg!(feature = "doxygen") {
+            writeln!(out, "  ///{}", line);
```

Comment:
Wrong indentation.

Elsewhere as well.

Reply:
What do you mean `wrong indentation`, the indentation is already checked by `rustfmt`.

Reply:
In the generated code, I want the number of spaces in front of the comment to be equal to the number of spaces in front of the thing that it is documenting, like it was prior to this PR.